### PR TITLE
last of the QC for existing food luas.

### PR DESCRIPTION
**Pull request by wrenffxi**
skipped a couple scripts for foods with mods not yet coded (skill-ups mostly)

### CODE REVIEW

File: scripts/globals/items/crepe_forestierre.lua [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+-----------------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+local result = 0;
```
Comment (TeoTwawki) [inline scripts/globals/items/crepe_forestierre.lua]: missing indents make Teo sad

File: scripts/globals/items/crepe_paysanne.lua [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+-----------------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    result = 0;
```
Comment (TeoTwawki) [inline scripts/globals/items/crepe_paysanne.lua]: local result

File: scripts/globals/items/ham_and_cheese_crepe.lua [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+-----------------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    result = 0;
```
Comment (TeoTwawki) [inline scripts/globals/items/ham_and_cheese_crepe.lua]: local here too

File: scripts/globals/items/kohlrouladen_+1.lua [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+-----------------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+local result = 0;
```
Comment (TeoTwawki) [inline scripts/globals/items/kohlrouladen_+1.lua]: indent

**Comment by wrenffxi** [comment]
Keep this open and I'll fix the whole folder.

**Comment by TeoTwawki** [comment]
@wrenffxi all finished? just making sure you weren't doing something still before merge

**Comment by wrenffxi** [comment]
All finished.

**Pull request merged**